src/main/java/com/example/demo/restservice/GreetingController.java

```diff
@@ -1,34 +1,101 @@
 package com.example.demo.restservice;
 
-import java.util.concurrent.atomic.AtomicLong;
-
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.sql.*;
+import java.util.concurrent.atomic.AtomicLong;
+
 
 @RestController
+@RequestMapping("/demo")
 public class GreetingController {
 
     private static final String template = "Hello, %s!";
     private static final String questTemplate = "%s ?";
     private final AtomicLong counter = new AtomicLong();
-
     public Imc test_imc;
+    public Secret secret;
+    public Connection conn;
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    @GetMapping("/pushsecret")
+    public Greeting pushSecret(@RequestParam(value = "secret", defaultValue = "mySecret") String pushedSecret,
+                               @RequestParam(value = "key", defaultValue = "mickey") String pushedKey) throws IOException {
+
+        this.createTable();
```

Comment:
You can trigger the createTableIfDoesntExist only once in your application life (then /pushsecret will be faster)